runtime/doc/lsp.txt

```diff
@@ -1732,15 +1736,15 @@ get_effective_tabstop({bufnr})          *vim.lsp.util.get_effective_tabstop()*
         (integer) indentation size
 
     See also: ~
-      • 'shiftwidth'
+      • shiftwidth
```

Comment:
why was this changed? the single-quotes are needed to hyperlink in :help

Reply:
a mistake forget restore.